PowerShellTools/Intellisense/AutoCompletionController.cs

```diff
@@ -177,6 +175,12 @@ private int ProcessKeystroke(uint nCmdID, IntPtr pvaIn)
             return VSConstants.S_FALSE;
         }
 
+        internal void SetAutoCompleteState(bool isAutoComplete)

```

Comment:
It's a state indicating last keystroke. It needs to be accessed to set so we can test different scenarios.